packages/cli/test/functional/test_site/testList.md

```diff
@@ -96,3 +103,102 @@
   * Sub-item B2 
   * Sub-item B3 {icon="./images/deer.jpg" i-width="200px" i-height="100px" i-class="text-warning"}
   * Sub-item B4
+
```

Comment:
Can you add a test case if the i-one-off is false? I think this currently won't work with your code

Reply:
Could you also add test cases showing that `the other attributes besides icon doesn't get inherited?